src/main.cpp

```diff
@@ -104,11 +106,13 @@ int main(int argc, char* argv[])
 
     auto context = xeus::make_context<zmq::context_t>();
 
+    setenv("R_HOME", "/Users/romainfrancois/miniforge3/lib/R", 1);
```

Comment:
This is temporary 😆

Reply:
We don't need to do this if we set it from outside, e.g. `R_HOME=<..> jupyter lab`